exercises/practice/grains/grains.coffee

```diff
@@ -0,0 +1,6 @@
+class Grains
+  @square: (number) ->
```

Comment:
Why are this "@" signs in front of the method? To pass the tests I had to remove them.

Reply:
It looks like the tests are not synchronized with this change.

Reply:
Could you elaborate on the tests not being synchronized? configlet isn't seeing any missing.

```
./bin/configlet sync --tests -e grains
Updating cached 'problem-specifications' data...
Checking exercises...
The `grains` exercise has up-to-date tests!
```

Reply:
> Why are this "@" signs in front of the methods?

That signifies they are  class methods so `Grains.square() and `new Grains.square()` both work. I'll investigate why Erik's change isn't working, but the example passed the CI.

Reply:
I opened up #249 so we can iterate there.